modules/local/epytope_peptide_prediction.nf

```diff
@@ -8,7 +8,7 @@ process EPYTOPE_PEPTIDE_PREDICTION {
 
     input:
     tuple val(meta), path(splitted), path(software_versions)
-    path netmhc_paths
+    val netmhc_paths
 
```

Comment:
Why did you use `val` here instead of `path`?

Reply:
Because this is a collection of tool paths of specified external prediction tools (or empty collection).